Formula/stubby.rb

```diff
@@ -0,0 +1,56 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver service based on getdns"
+  homepage "https://getdnsapi.net/blog/dns-privacy-daemon-stubby/"
+  url "https://github.com/getdnsapi/stubby/archive/v0.1.0.tar.gz"
+  sha256 "6210291850d6f7f124a5ba4bcee2f50814f020b7a0c4e67c6646bfe35ed5dd5b"
+  head "https://github.com/getdnsapi/stubby.git", :branch => "master"
+
+  link_overwrite "bin/stubby"
```

Comment:
Actually this won't work as `link_overwrite` will bail if the file belongs to another keg. I think the way we'll need to handle this is to upgrade getdns to a version that does not have stubby included so that there won't be a conflict.